torchgen/gen_vmap_plumbing.py

```diff
@@ -132,6 +132,14 @@ def is_mutated_arg(argument: Argument) -> bool:
     return argument.annotation is not None and argument.annotation.is_write
 
 
+def gen_check_escaped(check: str, what: str) -> str:
+    return f"""TORCH_CHECK(
+{check},
+"oops your{what} tensor escaped from vmap ",
+"See https://pytorch.org/functorch/stable/ux_limitations.html"
+  );"""
```

Comment:
this could be a problem with functorch internals or it could be an issue with vmap